app/components/Settings.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <Page actionBarHidden="true" class="page">
+    <FlexboxLayout style="flex: 1;"
+    flexDirection="column" id="container">
+      <TextView :text="'Settings'"
+      editable="false" id="settings-title" />
+
+      <StackLayout orientation="vertical" horizontalAlignment="center" style="padding: 10em;" id="settings-buttons">
+        <SpectraActionButton text="Change your goals" style="width: 80%" type="settings-orange" @tap="onGoalChange "/>
+
+        <SpectraActionButton text="Change your name" style="width: 80%" type="settings-green" @tap="onNameChange"/>
+
+        <SpectraActionButton text="View credits" style="width: 80%" type="settings-orange" @tap="onCredits"/>
+
+        <SpectraActionButton text="Share feedback" style="width: 80%" type="settings-green" @tap="onFeedback"/>
+
+        <SpectraActionButton text="Back" style="width: 50%" type="settings-orange" @tap="onBack"/>
+      </StackLayout>
+    </FlexboxLayout>
+  </Page>
+</template>
+
+<script>
+  import SpectraActionButton from "./UIControls/SpectraActionButton";
+  import App from "./App";
+  import SetAGoal from "./OnboardingScreens/SetAGoal";
+  var dialogs = require("tns-core-modules/ui/dialogs");
```

Comment:
Minor nit: It's better to use `const` rather than `var` for imports because it's unlikely that you'll reassign the variable.

Reply:
That makes sense!